src/status_im/common/standard_authentication/events.cljs

```diff
@@ -15,19 +15,14 @@
    (when on-auth-success (on-auth-success ""))))
 
 (defn authorize
-  [{:keys [db]} [{:keys [on-auth-success keycard-supported?] :as args}]]
+  [{:keys [db]} [{:keys [on-auth-success keycard-supported? theme] :as args}]]
```

Comment:
could you pls revert this change because this popup was removed already in my PR, thanks